client/checkout/platform-checkout/express-button/index.js

```diff
@@ -0,0 +1,35 @@
+/* global jQuery */
+/**
+ * External dependencies
+ */
+import ReactDOM from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import { WoopayExpressCheckoutButton } from './woopay-express-checkout-button';
+
+const renderPlatformCheckoutExpressButton = () => {
+	const platformCheckoutContainer = document.getElementById(
+		'wcpay-platform-checkout-button'
+	);
+
+	ReactDOM.render(
+		<WoopayExpressCheckoutButton
+			buttonSettings={ global.wcpayWooPayExpressParams.button }
+		/>,
+		platformCheckoutContainer
+	);
+};
+
+window.addEventListener( 'load', () => {
+	renderPlatformCheckoutExpressButton();
+} );
+
+// mount component again if parent fragment if re-rendered after ajax request by woocommerce core
+// https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/legacy/js/frontend/checkout.js#L372
+jQuery( function ( $ ) {
+	$( document ).ajaxComplete( function () {
+		renderPlatformCheckoutExpressButton();
+	} );
+} );
```

Comment:
I think this part is not needed. In the [save-user checkbox section](https://github.com/Automattic/woocommerce-payments/blob/add/1427-style-woopay-express-checkout-button/client/checkout/platform-checkout/index.js#L66) we had to add this extra check because the save user section is pushed to the document from frontend. And after this ajax call it would disappear from the screen without this extra jquery check.
As we are preparing the placeholder `div` for the express button from the backend, this check might be unnecessary.

For me, the button loads perfectly if I remove these codes too. But you may want to verify it on your own as the ajax request order could be different for us.

Reply:
Resolved in ef6cb212